src/components/login-view/login-view.jsx

```diff
@@ -1,15 +1,96 @@
-export const LoginView = () => {
+import React from "react";
+import { useState } from "react";
+import PropTypes from "prop-types";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+
+export const LoginView = ({onLoggedIn}) => {
+  const [username, setUsername] = useState ("");
+  const [password, setPassword] = useState ("");
+  const loginURL = "https://myquickmovieapi.onrender.com/login"
+
+  const handleSubmit = (event) => {
+    // this prevents the default behavior of the form which is to reload the entire page
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password
+    };
+   
+    fetch(loginURL, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log("Login response: ", data);
+        if (data.user) {
+          localStorage.setItem("user", JSON.stringify(data.user));
+          localStorage.setItem("token", data.token);
+          onLoggedIn(data.user, data.token);
```

Comment:
Check the comment on main-view first. :warning: 
A. The function is called with two argument. The second will be ignored.